lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -553,6 +553,81 @@ else
 
 end
 
+e2function number entity:canTool(entity target, string toolname)
```

Comment:
I think `target` should be replaced with `player` to avoid confusion.

Reply:
`entity` is actually the player you're checking can use the `toolname` tool on `target`. `target` can be any entity class.